### PR TITLE
Improve distribution docs - continued

### PR DESCRIPTION
**Pull request by tblazina**
This PR is a continuation of PR ##3474 and would hopefully close issue #2992. I've added the doc strings to public methods for distribution classes in `continuous.py`, `discrete.py` etc. As a relative newcomer to the project and being not totally familiar with all the different distributions, I wasn't sure if the wording is correct for some of the docstrings (e.g. the logp and random methods for the LKJCholeskyCov class in `multivariate.py`).

### CODE REVIEW

File: pymc3/distributions/continuous.py [hunk 1]
```diff
@@ -233,6 +233,20 @@ def _repr_latex_(self, name=None, dist=None):
...
 
     def logcdf(self, value):
+        """
+        Compute the log of the cumulative distribution function for Uniform distribution
+        at the specified value.
+
+        Parameters
+        ----------
+        value: numeric
+            Value(s) for which log CDF is calculate. If the log CDF for multiple
```
Comment (ColCarroll) [inline pymc3/distributions/continuous.py]:
```suggestion
            Value(s) for which log CDF is calculated. If the log CDF for multiple
```

File: pymc3/distributions/continuous.py [hunk 2]
```diff
@@ -290,6 +304,20 @@ def _repr_latex_(self, name=None, dist=None):
...
 
     def logcdf(self, value):
+        """
+        Compute the log of the cumulative distribution function for Flat distribution
+        at the specified value.
+
+        Parameters
+        ----------
+        value: numeric
+            Value(s) for which log CDF is calculate. If the log CDF for multiple
```
Comment (ColCarroll) [inline pymc3/distributions/continuous.py]:
```suggestion
            Value(s) for which log CDF is calculated. If the log CDF for multiple
```

File: pymc3/distributions/continuous.py [hunk 3]
```diff
@@ -343,6 +371,20 @@ def _repr_latex_(self, name=None, dist=None):
...
 
     def logcdf(self, value):
+        """
+        Compute the log of the cumulative distribution function for HalfFlat distribution
+        at the specified value.
+
+        Parameters
+        ----------
+        value: numeric
+            Value(s) for which log CDF is calculate. If the log CDF for multiple
```
Comment (ColCarroll) [inline pymc3/distributions/continuous.py]:
```suggestion
            Value(s) for which log CDF is calculated. If the log CDF for multiple
```

File: pymc3/distributions/continuous.py [hunk 4]
```diff
@@ -487,6 +529,20 @@ def _repr_latex_(self, name=None, dist=None):
...
 
     def logcdf(self, value):
+        """
+        Compute the log of the cumulative distribution function for Normal distribution
+        at the specified value.
+
+        Parameters
+        ----------
+        value: numeric
+            Value(s) for which log CDF is calculate. If the log CDF for multiple
```
Comment (ColCarroll) [inline pymc3/distributions/continuous.py]:
```suggestion
            Value(s) for which log CDF is calculated. If the log CDF for multiple
```

**Review by ColCarroll — COMMENTED** [review]
this is wonderful! there are a few more of that one typo, but this is a great contribution.

**Comment by ColCarroll** [comment]
Thanks again! I'll merge after tests pass (doesn't _look_ like any code changed, but good practice anyways)

**Comment by tblazina** [comment]
Cheers! Yes, always good to rerun tests even without code changes, then you catch dumb mistakes like accidentally deleting a variable definition when cutting and pasting doc strings 🤦‍♂

**Comment by twiecki** [comment]
Thanks @tblazina !

**Pull request merged**